### PR TITLE
reworked tutorial to fit current needs.

### PR DESCRIPTION
**Pull request by LukasReindlGit**
Updated tutorial to new needs

### CODE REVIEW

File: Assets/Scripts/DayNight/CustomLightSetupMenu.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            Clear();
+            LightSetupController lightSetupController = LightSetupController.Instance;
+
+            if (lightSetupController == null)
+            {
+                return;
+            }
+
+            // TODO Chapter 8: Add a button for each light setup to the menu:
+            // Use Add(new ButtonItem(resourceIcon, (controller)=> { // Do something. })));
```
Comment (tomwim) [inline Assets/Scripts/DayNight/CustomLightSetupMenu.cs]:
```suggestion
            // Use Add(new ButtonItem(resourceIcon, (controller) => { // Do something. })));
```

File: Assets/Scripts/DayNight/CustomLightSetupMenu.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        }
+
+
+        private void SetLightSetup(int lightSetupIndex)
+        {
+            // TODO Chapter 8: Set the light setup in the LightSetupController.Instance:
+
+            // TODO Chapter 11: Replace the instance call with a LightSetupCommand. Make sure to invalidate previous lightCommands if available before executing the new one.
+
```
Comment (tomwim) [inline Assets/Scripts/DayNight/CustomLightSetupMenu.cs]: Empty line

File: Assets/Scripts/Flashlight/Battery/BatteryNetworking.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    {
+        private Battery battery;
+        private PhotonView photonView;
+
+        private void OnEnable()
+        {
+            battery = GetComponent<Battery>();
+            photonView = GetComponent<PhotonView>();
+
+            // TODO Chapter 11: Add this component to the ObservedComponents list in the photonview.
```
Comment (tomwim) [inline Assets/Scripts/Flashlight/Battery/BatteryNetworking.cs]:
```suggestion
            // TODO Chapter 11: Add this component to the ObservedComponents list in the PhotonView.
```

File: Assets/Scripts/Flashlight/Battery/BatteryPersistenceData.cs [hunk 4]
```diff
@@ -0,0 +1,21 @@
+﻿using System.Runtime.Serialization;
+using Innoactive.Hub.Persistence;
+
+namespace Innoactive.HubTutorial
+{
+    [DataContract]
+    public class BatteryPersistenceData : PersistentProperty.PropData
```
Comment (tomwim) [inline Assets/Scripts/Flashlight/Battery/BatteryPersistenceData.cs:10]: Do we need docu here? Might make sense for the tutorial?

File: Assets/Scripts/Flashlight/Battery/BatteryPersistenceTranslator.cs [hunk 5]
```diff
@@ -0,0 +1,37 @@
+﻿using Innoactive.Hub.Persistence.Unity;
+using UnityEngine;
+
+namespace Innoactive.HubTutorial
+{
+    public class BatteryPersistenceTranslator : SingleComponentPropertyTranslator<BatteryPersistenceData, Battery>
```
Comment (tomwim) [inline Assets/Scripts/Flashlight/Battery/BatteryPersistenceTranslator.cs:9]: Do we need docu here? Might make sense for the tutorial?

File: Assets/Scripts/Flashlight/FlashlightContextMenu.cs [hunk 6]
```diff
@@ -40,6 +39,8 @@ protected virtual void Awake()
         /// <returns>List of buttons to add</returns>
         public List<IMenuItem> ProvideContextMenuActions()
         {
+         ContextMenuItemDesign design = new ContextMenuItemDesign();
```
Comment (tomwim) [inline Assets/Scripts/Flashlight/FlashlightContextMenu.cs]: wrong indentation

File: Assets/Scripts/Flashlight/FlashlightNetworking.cs [hunk 7]
```diff
@@ -91,16 +53,19 @@ private void FlashlightSpreadChanged(object sender, Flashlight.SpreadChangedEven
         /// <param name="args">Arguments which hold information about the current state</param>
         private void FlashlightStateChanged(object sender, Flashlight.LightStateChangedEventArgs args)
         {
-            // TODO Chapter 11: Change state of light for all other users using HubCommand.
+            // TODO Chapter 11: Change state of light for all other users using the photonView RPC.
+
```
Comment (tomwim) [inline Assets/Scripts/Flashlight/FlashlightNetworking.cs]: empty line

File: Assets/Scripts/Flashlight/FlashlightPeristenceTranslator.cs [hunk 8]
```diff
@@ -1,11 +1,12 @@
...
 
-namespace HubTutorial
+namespace Innoactive.HubTutorial
 {
     /// <summary>
     /// Component that translates component representing a <see cref="Flashlight"/>.
     /// </summary>
-    public class FlashlightPersistenceTranslator : SingleComponentPropertyTranslator<FlashlightPersistenceData, Flashlight>
+    public class
+        FlashlightPersistenceTranslator : SingleComponentPropertyTranslator<FlashlightPersistenceData, Flashlight>
```
Comment (tomwim) [inline Assets/Scripts/Flashlight/FlashlightPeristenceTranslator.cs]: weird line break

**Review by tomwim — CHANGES_REQUESTED** [review]
Only code review, havent checked project out yet

File: Assets/Scripts/Flashlight/FlashlightPeristenceTranslator.cs (hunk 8 above)
Comment (LukasReindlGit) [reply Assets/Scripts/Flashlight/FlashlightPeristenceTranslator.cs]: wtf?

**Review by tomwim — APPROVED** [review]

**Pull request merged**